src/main/scala/cromwell/Main.scala

```diff
@@ -20,16 +21,39 @@ object Actions extends Enumeration {
 }
 
 object Main extends App {
-  val Properties = System.getProperties
-  val LoggerProperty = "CROMWELL_LOGGER"
-  lazy val Log = LoggerFactory.getLogger("main")
-
-  Option(Properties.getProperty(LoggerProperty)) match {
-    case None if args.headOption.map {_.toUpperCase}.contains("SERVER") =>
-      Properties.setProperty(LoggerProperty, "SERVER")
-    case _ =>
+  def initLogging() = {
+    val LogRootDefault = "."
+    val LogModeDefault = "CONSOLE"
+    val LogLevelDefault = "INFO"
+
+    val logRoot = Option(System.getProperty("LOG_ROOT")) match {
+      case None =>
+        System.setProperty("LOG_ROOT", LogRootDefault)
+        LogRootDefault
+      case Some(x) => x
+    }
+
+    if (Option(System.getProperty("LOG_MODE")).isEmpty)
+      System.setProperty("LOG_MODE", LogModeDefault)
+
+    if (Option(System.getProperty("LOG_LEVEL")).isEmpty)
+      System.setProperty("LOG_LEVEL", LogLevelDefault)
+
+    try {
+      Files.createDirectories(Paths.get(logRoot))
+    } catch {
+      case e: FileAlreadyExistsException =>
+      case e: Throwable =>
+        System.err.println(s"Could not create log directory: $logRoot")
+        e.printStackTrace()
+        System.exit(1)
+    }
   }
 
+  initLogging()
```

Comment:
I had it all in the constructor initially, then Jeff thought it'd be better in a function, and I personally don't care all that much and didn't have a compelling reason to keep it all in the constructor